tests/dependency/post_build.rs

```diff
@@ -3,13 +3,15 @@ use std::{path::{Path, PathBuf}, env};
 use example as _;
 
 fn main() {
+    let current_dir = env::current_dir().unwrap();
+    let current_parent = current_dir.parent().unwrap();
     assert_eq!(env::var("CRATE_BUILD_COMMAND").unwrap(), "cargo build --package dependency");
-    assert_eq!(PathBuf::from(env::var("CRATE_MANIFEST_DIR").unwrap()), Path::new(".").canonicalize().unwrap());
-    assert_eq!(PathBuf::from(env::var("CRATE_MANIFEST_PATH").unwrap()), Path::new("Cargo.toml").canonicalize().unwrap());
+    assert_eq!(PathBuf::from(env::var("CRATE_MANIFEST_DIR").unwrap()), current_dir);
+    assert_eq!(PathBuf::from(env::var("CRATE_MANIFEST_PATH").unwrap()), current_dir.join("Cargo.toml"));
     assert_eq!(env::var("CRATE_PROFILE").unwrap(), "debug");
     assert_eq!(env::var("CRATE_TARGET").unwrap(), "");
     assert_eq!(env::var("CRATE_TARGET_TRIPLE").unwrap(), "");
-    assert_eq!(PathBuf::from(env::var("CRATE_TARGET_DIR").unwrap()), Path::new("../target").canonicalize().unwrap());
-    assert_eq!(PathBuf::from(env::var("CRATE_OUT_DIR").unwrap()), Path::new("../target").join("debug").canonicalize().unwrap());
+    assert_eq!(PathBuf::from(env::var("CRATE_TARGET_DIR").unwrap()), current_parent.join("target"));
+    assert_eq!(PathBuf::from(env::var("CRATE_OUT_DIR").unwrap()), current_parent.join("target/debug"));
```

Comment:
Forward slashes are not a valid directory separator when using UNC-style paths on windows. So it's better to use `join` multiple times instead:

```suggestion
    assert_eq!(PathBuf::from(env::var("CRATE_OUT_DIR").unwrap()), current_parent.join("target").join("debug"));
```

Reply:
Makes sense